packages/components/src/Tokens/Tokens.tsx

```diff
@@ -27,14 +31,18 @@ export class UserTokenAccount extends React.Component<{}, UserTokenAccountStates
   }
 
   public render(): JSX.Element | null {
+    const { user } = this.props;
+
+    const accountSignupStep = this.getUserStep(user);
+
     return (
       <TokenAccountOuter>
         <TokenAccountInner>
           <UserTokenAccountHeader />
           <FlexColumns>
             <FlexColumnsPrimary>
               <FlexColumnsPrimaryModule>
-                <UserTokenAccountSignup step={"completed"} />
+                <UserTokenAccountSignup step={accountSignupStep} />
```

Comment:
I updated this flow a little bit in a new branch so we're going to have some merge conflicts to work out. Should be easy fixes but there are a few things we should discuss. Same with `TokensAccountSignup.tsx`